index.html

```diff
@@ -762,18 +838,126 @@ <h2><dfn>CaptureController</dfn></h2>
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\focusWindowClosed]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot tracks explicit closure of the window of opportunity, which happens when
+                      {{CaptureController/setFocusBehavior()}} is called, or the task to close the window executes.
+                      As applications can easily avoid such errors, {{CaptureController/setFocusBehavior()}} when
+                      this slot is set to <code>true</code> throw.
+                    </div>
+                  </p>
+                </li>
+              </ol>
+            </dd>
+            <dt>
+              <dfn>setFocusBehavior</dfn>
+            </dt>
             <dd>
               <p>
-                Create one internal slot: <dfn data-dfn-for="CaptureController">[[\isBound]]</dfn>,
-                initialized to <code>false</code>.
+                Run the following steps:
               </p>
+              <ol>
+                <li>
+                  <p>
+                    If {{CaptureController/[[isBound]]}} is <code>false</code>,
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[videoTrack]]}} is <code>null</code>,
+                    [=exception/throw=] a "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[videoTrack]]}}.{{MediaStreamTrack/getSettings()}}.{{displaySurface}}
+                    is neither {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    <!-- TODO: Fix the ReadyState reference. -->
+                    If {{CaptureController/[[videoTrack]]}}'s
+                    <code>ReadyState</code> is NOT {{MediaStreamTrackState/"live"}},
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[focusWindowClosed]]}} is <code>true</code>,
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[initialFocusLost]]}} is <code>true</code>,
```

Comment:
At this step, we might want to run steps in parallel, given this might actually happen in another process.

Reply:
To be sure I understand you correctly, could you clarify the proposed change?

Reply:
Ping @youennf for this question.